src/task1.ts

```diff
@@ -0,0 +1,7 @@
+let inString = process.openStdin();
+
+console.log('Type any string...');
+
+const revertString = (str: string): string => str.toString().split('').reverse().join('').trim();
```

Comment:
maybe `reverseString` could have been more semantic, since `revert` I think has a bit different meaning. But it's inconsequential comment, just a note